test_regress/t/t_inside_nonint.out

```diff
@@ -0,0 +1,9 @@
+CALL 1:
+string: WO
+real: 0.000000
```

Comment:
Please have the test check for correct values directly (either with `checks macro or direct "if" check - see other tests), rather than use a "golden result" file.  This makes it easier to check on other simulators, thanks.

Reply:
Done. The regression test has been changed to use "if" checks.